### ISSUE TITLE
Deleting users after failing to login

### ISSUE THREAD

**Issue by gregory** [t3db0t/meteor-accounts-invite#5]
For anyone using this, just wanted to point out that this lib will delete any user that [fail to login, and doesn't have user.services.accountsInvite](https://github.com/t3db0t/meteor-accounts-invite/blob/master/server/accounts-invite-server.js#L41)

I'm glad i tested this before going to prod.

**Comment by t3db0t** [comment]
This is true, it's designed to prevent a bunch of unused user records from sticking around... are you saying that this is deleting users in an undesired way? Can you explain the scenario?  Thank you for reporting!

**Comment by gregory** [comment]
yes that's what i'm saying :) 
I was logged in, installed your package, and because my user didn't get accountsInvite, it removed it from the collection.
